tox.ini

```diff
@@ -139,3 +139,6 @@ deps = {[testenv:docs]deps}
        oslosphinx
 commands =
     pifpaf -g GNOCCHI_INDEXER_URL run postgresql -- sphinx-versioning build doc/source doc/build/html
+
+[doc8]
+ignore-path = doc/source/rest.rst,doc/source/comparison-table.rst
```

Comment:
This is to ignore doc8 on rest.rst (autogenerated from the j2 template, as we do currently) and comparison-table, which is a very large table that doc8 does not like because lines are too long.